airflow/jobs.py

```diff
@@ -570,8 +570,8 @@ def prioritize_queued(self, session, executor, dagbag):
                     self.logger.info("Pickling DAG {}".format(dag))
                     pickle_id = dag.pickle(session).id
 
-                if dag.dag_id in overloaded_dags or dag.concurrency_reached:
-                    overloaded_dags.add(dag.dag_id)
+                if dag.dag_id in dag_blacklist or dag.concurrency_reached:
```

Comment:
that's for the `or concurrency_reached`, I can break it down as an elif, though `dag_blacklist` is a set so it won't grow anyways